core/slice.go

```diff
@@ -354,33 +348,178 @@ func (sl *Slice) Append(header *types.WorkObject, domTerminus common.Hash, domOr
 				}
 			}
 
-			// Compute the cubic flow discount
-			discountedConversionAmount := misc.ApplyCubicDiscount(conversionAmountInQuai, header.ConversionFlowAmount())
-			discountedConversionAmountInInt, _ := discountedConversionAmount.Int(nil)
+			// sort the newInboundEtxs based on the decreasing order of the max slips
+			sort.SliceStable(newInboundEtxs, func(i, j int) bool {
+				etxi := newInboundEtxs[i]
+				etxj := newInboundEtxs[j]
+
+				var slipi *big.Int
+				if etxi.EtxType() == types.ConversionType {
+					// If no slip is mentioned it is set to 90%
+					slipAmount := new(big.Int).Set(params.MaxSlip)
+					if len(etxi.Data()) > 1 {
```

Comment:
So we can't set a 1-byte slip right? Need to be careful with that for quais/docs - if someone sets a 1-byte slip they could get rekt

Reply:
yeah, its two bytes, i agree, i made a note now to make sure we implement it that way in quais and also in the docs